pkg/spec/spec.go

```diff
@@ -45,12 +45,15 @@ func GroupFromBytes(b []byte) (*Group, error) {
 			return nil, fmt.Errorf("failed to unmarshal spec item %w: \n%s", err, specItem)
 		}
 
+		kind := strings.ToLower(s.Kind)
+
 		if s.Spec == nil {
-			return nil, fmt.Errorf("no spec field on %q document", s.Kind)
+			if kind == "" {
+				return nil, errors.New(`Missing required fields ("spec", "kind") on provided configuration.`)
+			}
+			return nil, fmt.Errorf(`Missing required fields ("spec") on provided %q configuration.`, s.Kind)
```

Comment:
this is an ad-hoc request in the Figma that I missed when building the CLI

![image](https://github.com/fleetdm/fleet/assets/4419992/300b3215-7c45-4209-a036-371df7c7499e)